### PR TITLE
Tier 1 tutorial: providing prebuilt binary for STM32 WiFi

### PR DESCRIPTION
**Pull request by atigyi**
[ Description ]
This PR contains a prebuilt binary / firmware for STM32F401RE + WiFi IDW01M1 + MEMS IKS01A1 sandwich board. Coupled with this binary a Tier 1 tutorial is released which is the simplest tutorial we can provide now: containing no compilation or linkage steps, just flashing the device then serial terminal provisioning for WiFi and Xively and then booom connected to Xively.

[ JIRA ]
T1 Tutorial for STM32F4xx WiFi board
https://developer.xively.com/docs/stm32f4xx-nucleo-quick

[ Reviewer ]

[ QA ]
Went through the T1 tutorial multiple times, works at my computer. This review should include the revision of the tutorial, going through it step-by-step. Thank you. https://developer.xively.com/docs/stm32f4xx-nucleo-quick

[ Release Notes ]
The repo is extended with a prebuilt binary for STM32F401RE + WiFi IDW01M1 + MEMS IKS01A1. Tier 1 tutorial is also provided.

### CODE REVIEW

File: examples/log.txt [hunk 1]
```diff
@@ -1,3 +0,0 @@
-/Applications/Xcode.app/Contents/Developer/usr/bin/make -C /Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example all
-[cc] /Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example/bin/mqtt_logic_example
-cc /Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example/obj/common/commandline.o /Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example/obj/mqtt_logic_example.o -L/Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example/../../obj/osx  -lxively -lm -lpthread -L  -lwolfssl -o /Users/ohumenczuk/workspace/xively/libxively-v3/xi_client_c/examples/mqtt_logic_example/bin/mqtt_logic_example
```
Comment (Palantir555) [inline examples/log.txt]: I'm curious, what's this file??

File: examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c [hunk 2]
```diff
@@ -763,14 +763,17 @@ static inline void print_user_config_debug_banner( void )
...
     printf( "\r\n|               --------------------------               |" );
-    printf( "\r\n| WARNING: Do NOT copy-paste long strings to your serial |" );
-    printf( "\r\n| -------  terminal. Only strings under ~6 characters    |" );
-    printf( "\r\n|          can be copied with medium/high reliability    |" );
+    printf( "\r\n| WARNING: Copy-pasting long strings to your serial      |" );
+    printf( "\r\n| -------  terminal may end up in character losses.      |" );
+    printf( "\r\n|          To solve this issue introduce character       |" );
+    printf( "\r\n|          transmit delay in the terminal settings.      |" );
+    printf( "\r\n|          coolTerm:                                     |" );
+    printf( "\r\n|   Options->Transmit->Use transmit character delay      |" );
```
Comment (Palantir555) [inline examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c]: :+1: Good change.

Maybe we can reword "may end up in character losses" to "may result in the loss of characters", "introduce character transmit delay" to "use a delay between characters" and "coolTerm:" to "e.g. In coolTerm:".

Just my 2 cents; I'm certainly not the right person correct English grammar :P

**Comment by Palantir555** [comment]
Reviewed, tested, good to merge :+1:

File: examples/log.txt (hunk 1 above)
Comment (atigyi) [inline examples/log.txt]: This was checked in by Olgierd, my guess is this was accidental. Probably he redirected the build output into log.txt and then checked this in.

File: examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c (hunk 2 above)
Comment (atigyi) [inline examples/stm32/xively_demo_stm32f4xx_nucleo_wifi/Src/main.c]: ok, I'll change these.

**Pull request merged**